packages/contentstack-utilities/src/contentstack-management-sdk.ts

```diff
@@ -105,6 +104,11 @@ class ManagementSDKInitiator {
       }
     }
 
+    const earlyAccessHeaders = configStore.get(`earlyAccessHeaders`);
+    if (Object.keys(earlyAccessHeaders).length > 0) {
+      option.early_access = Object.values(earlyAccessHeaders);
+    }
```

Comment:
Handle the case when `earlyAccessHeaders` is not part of the existing config/does not exist. In such a case, the current code will throw an error/fail.

Reply:
done, thanks